bindata/network/kuryr/001-rbac.yaml

```diff
@@ -58,7 +58,13 @@ rules:
   - securitycontextconstraints
   verbs:
   - use
-
+- apiGroups: ["machine.openshift.io"]
+  resources:
+  - machines
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Are the verbs get and watch needed even when only List is used?

Reply:
ah, I just saw this will be used on Kuryr itself. Disregard this comment.

Reply:
We're actually doing watch (it's for Kuryr). I'll check if it's possible to get rid of get and list one.

Reply:
On the other head it's just `get` and `list`, nothing too crazy here. I'll leave those in case in the future Kuryr will actually need to list them.